js/utils/notification.js

```diff
@@ -65,12 +66,19 @@ function playNotifSound() {
 }
 
 export function launchNativeNotification(notifTitle = '', options = {}) {
+  if (options.body !== undefined && typeof options.body !== 'string') {
+    throw new Error('If providing a notification body, it must be provided ' +
+      'as a string.');
+  }
+
   const notifOptions = {
     silent: true,
     ...(_.omit(options || {}, 'onclick', 'onerror')),
+    body: options.body ? stripHtml(options.body) : '',
   };
 
-  const notif = new Notification(notifTitle, notifOptions);
+  const title = typeof notifTitle === 'string' ? stripHtml(notifTitle) : notifTitle;
```

Comment:
Yes, the native notification will throw an error. I didn't see any need for our wrapper function to throw the error instead.

Reply:
Wouldn't the native notification also throw an error on body? Why check only that?

Reply:
stripHtml would throw the error and for whatever reason I though it would be confusing for the user to get an error from stripHtml here, whereas I didn't think the native notification throwing the error would be confusing.

Since it seems to be an important enough thing for you, I've updated things.